itest/list_on_test.go

```diff
@@ -670,4 +666,5 @@ func init() {
 	allTestCases = append(allTestCases, channelFeePolicyTestCases...)
 	allTestCases = append(allTestCases, walletImportAccountTestCases...)
 	allTestCases = append(allTestCases, basicFundingTestCases...)
+	allTestCases = append(allTestCases, sendToRouteTestCases...)
```

Comment:
For all the tests that we add to the list here, should we add a static prefix to their names here and then have more short names in the actual slice where they're defined? Would perhaps help to locate them a bit more easily?
Example (dummy code):

```
allTestCases = appendPrefixed(allTestCases, "send to route", sendToRouteTestCases...)


...


var sendToRouteTestCases = []*lntest.TestCase{
	{
		Name: "single hop sync",

	},
	{
		Name: "single hop stream",

	},
	{
		Name: "single hop v2",

	},
}
```

But just an idea and non-blocking.

Reply:
yeah i like it - added the suggested change in a followup commit